tests/unit-tests/test-class-modules.php

```diff
@@ -200,4 +201,97 @@ public function testCourseModulesColumnShouldNotShowMoreLinkIfLessThan4Modules()
 		$this->assertNotContains( 'more', $column_output, 'The "more" link shouldn\'t be present.' );
 	}
 
+	public function testModuleTeacherMeta_WhenAddedToACourse_TeacherIdGetsAddedToMeta() {
+		/* Arrange */
+		$this->login_as_teacher();
+
+		$course = $this->factory->get_course_with_lessons(
+			[
+				'module_count'   => 0,
+				'lesson_count'   => 1,
+				'question_count' => 0,
+			]
+		);
+
+		$module = wp_insert_term(
+			'Get Started',
+			'module',
+			array(
+				'description' => 'A yummy apple.',
+				'slug'        => 'get-started',
+			)
+		);
+
+		/* Act */
+		wp_set_object_terms( $course['course_id'], [ $module['term_id'] ], 'module' );
+
+		/* Assert */
+		$this->assertSame( absint( get_term_meta( $module['term_id'], 'module_author', true ) ), wp_get_current_user()->ID );
+	}
+
+	public function testModuleTeacherMeta_WhenRemovedFromACourse_TeacherIdGetsRemovedFromMeta() {
+		/* Arrange */
+		$this->login_as_teacher();
+
+		$course = $this->factory->get_course_with_lessons(
+			[
+				'module_count'   => 0,
+				'lesson_count'   => 1,
+				'question_count' => 0,
+			]
+		);
+
+		$module = wp_insert_term(
+			'Get Started',
+			'module',
+			array(
+				'description' => 'A yummy apple.',
+				'slug'        => 'get-started',
+			)
+		);
+
+		wp_set_object_terms( $course['course_id'], [ $module['term_id'] ], 'module' );
+
+		/* Act */
+		wp_remove_object_terms( $course['course_id'], $module['term_id'], 'module' );
+
+		/* Assert */
+		$this->assertSame( get_term_meta( $module['term_id'], 'module_author', true ), '' );
```

Comment:
I think expected and actual values should we swapped here (in other tests probably too).
First argument is an expected value, the second one is actual.

Reply:
Good catch, Thanks! Updated [here](https://github.com/Automattic/sensei/pull/5376/commits/8098d74879ecff1149180b65ede9b790cf380a66)